boards/riscv32/m2gl025_miv/doc/index.rst

```diff
@@ -19,12 +19,8 @@ Building
 
 Applications for the ``m2gl025_miv`` board configuration can be built as usual
 (see :ref:`build_an_application`).
-In order to build the application for ``m2gl025_miv``, set the ``BOARD`` variable
-to ``m2gl025_miv``.
-
-.. code-block:: bash
-
-   export BOARD="m2gl025_miv"
+In order to build the application for ``m2gl025_miv``, set the ``-DBOARD``
```

Comment:
Same here as with hifive; needs `--`; perhaps a link to the relevant `west  build` docs would help?

Reply:
I will rework this and the previous one

Reply:
Fixed